shared/actions/push.js

```diff
@@ -125,6 +131,16 @@ function* pushNotificationSaga(notification: PushGen.NotificationPayload): Saga.
         // ignore it
         break
       }
+      // If a boxed message is attached to the notification, unbox.
+      if (payload.m) {
+        logger.info('Push notification: unboxing notification message')
+        yield Saga.call(RPCChatTypes.localUnboxMobilePushNotificationRpcPromise, {
```

Comment:
likely shouldn't do this if no convID or payload

Reply:
done